src/parser.rs

```diff
@@ -75,6 +77,15 @@ impl fmt::Display for ParseError {
             ParseError::UnexpectedEOFWanted(kinds) => {
                 write!(f, "unexpected end of file, wanted any of {:?}", kinds)
             }
+            ParseError::UnexpectedDuplicatedArgs(range, ident) => {
+                write!(
+                    f,
+                    "argument `{}' is duplicated in {}..{}",
+                    ident,
+                    usize::from(range.start()),
+                    usize::from(range.end())
```

Comment:
I believe the `Debug` implementation of a `TextRange` produces the same string:
```suggestion
                    "argument `{}` is duplicated in {:?}",
                    ident,
                    range
```

Also, I think that single quote (after the first `{}`) is supposed to be a backtick.

Reply:
This doesn't compile then.

Reply:
Hmm, this does seem to work as expected:
```rust
                write!(
                    f,
                    "argument `{}` is duplicated in {:?}",
                    ident,
                    range
                )
```
Anyhow, the change is not really significant, I think the current state is fine as well.